python/lsst/ts/scheduler/coldstart/coldstarter.py

```diff
@@ -0,0 +1,24 @@
+import sqlite3
+from database import *
+
+
+
+def get_observation_history():
```

Comment:
Not sure this method is actually required... This could be part of the initialization script.